### PR TITLE
[Fix] Update the cn doc

### PR DESCRIPTION
**Pull request by lianqing01**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

update the getting_started in cn doc.

### CODE REVIEW

File: docs/zh_cn/faq.md [hunk 1]
```diff
@@ -4,6 +4,36 @@
 
 ## MMCV/MMDet/MMDet3D Installation
 
+- 跟MMCV, MMDetection, MMSegmentation and MMDection3D相关的编译问题; "ConvWS is already registered in conv layer"; "AssertionError: MMCV==xxx is used but incompatible. Please install mmcv>=xxx, \<=xxx."
```
Comment (Tai-Wang) [inline docs/zh_cn/faq.md:7]: add space between chinese and english characters like
跟 MMCV, MMDetection xxxxx, MMDetection3D 相关的编译问题

File: docs/zh_cn/getting_started.md [hunk 2]
```diff
@@ -1,144 +1,77 @@
 # 依赖
 
-- Linux 或者 macOS (实验性支持 Windows)
+MMDection3D 可以安装在 Linux, MacOS, (实验性支持 Windows)的平台上，它具体需要下列安装包:
```
Comment (Tai-Wang) [inline docs/zh_cn/getting_started.md:3]: (实验性支持 Windows) 的平台上

File: docs/zh_cn/getting_started.md [hunk 3]
```diff
@@ -184,58 +117,27 @@ pip install -v -e .  # or "python setup.py develop"
...
-```
-
-## 从零开始的安装脚本
-
-以下是一个基于 conda 安装 MMdetection3D 的脚本
-
-```shell
-conda create -n open-mmlab python=3.7 -y
-conda activate open-mmlab
+   我们已经支持 spconv2.0. 如果用户已经安装spconv 2.0， 代码会默认使用spconv 2.0。它可以比原生mmcv spconv 使用更少的内存。 用户可以使用下列的命令来安装spconv 2.0.
```
Comment (Tai-Wang) [inline docs/zh_cn/getting_started.md:120]: Please note the usage of space and punctuation
我们已经支持 spconv 2.0。如果用户已经安装 spconv 2.0，代码会默认使用 spconv 2.0。它可以比原生 mmcv 中的 spconv 使用更少的内存。用户可以使用下列的命令来安装 spconv 2.0：

File: docs/zh_cn/getting_started.md [hunk 4]
```diff
@@ -184,58 +117,27 @@ pip install -v -e .  # or "python setup.py develop"
...
-pip install mmcv-full
+   xxx 表示  CUDA 的版本。
 
-# 安装 mmdetection
-pip install git+https://github.com/open-mmlab/mmdetection.git
+   例如, 使用 CUDA 10.2, 对应命令是  `pip install cumm-cu102 && pip install spconv-cu102`.
 
-# 安装 mmsegmentation
-pip install git+https://github.com/open-mmlab/mmsegmentation.git
+   支持的CUDA 版本包括 10.2, 11.1, 11.3, and 11.4. 用户可以通过源码编译来在这些版本上安装. 具体细节请参考 [spconv v2.x](https://github.com/traveller59/spconv).
```
Comment (Tai-Wang) [inline docs/zh_cn/getting_started.md:131]: 支持的 CUDA 版本 xxxx 11.3 和 11.4。

File: docs/zh_cn/faq.md [hunk 5]
```diff
@@ -4,6 +4,36 @@
 
 ## MMCV/MMDet/MMDet3D Installation
 
+- 跟 MMCV, MMDetection, MMSegmentation and MMDection3D相关的编译问题; "ConvWS is already registered in conv layer"; "AssertionError: MMCV==xxx is used but incompatible. Please install mmcv>=xxx, \<=xxx."
```
Comment (Tai-Wang) [inline docs/zh_cn/faq.md:7]: 跟 MMCV, MMDetection, MMSegmentation 和 MMDetection3D 相关的编译问题

File: docs/zh_cn/faq.md [hunk 6]
```diff
@@ -4,6 +4,36 @@
...
+  MMDETECTION3D需要的 MMCV, MMDetection 和 MMSegmentation的版本列在了下面。请安装正确版本的 MMCV、MMDetection 和 MMSegmentation 以避免相关的安装问题。
```
Comment (Tai-Wang) [inline docs/zh_cn/faq.md:8]: MMDetection3D 需要的

Comment (Tai-Wang) [reply docs/zh_cn/faq.md:8]: MMSegmentation 的版本

File: docs/zh_cn/getting_started.md [hunk 7]
```diff
@@ -1,144 +1,77 @@
 # 依赖
 
-- Linux 或者 macOS (实验性支持 Windows)
+MMDection3D 可以安装在 Linux, MacOS, (实验性支持 Windows) 的平台上，它具体需要下列安装包:
```
Comment (Tai-Wang) [inline docs/zh_cn/getting_started.md:3]: MMDetection3D

**Review by Tai-Wang — APPROVED** [review]

**Comment by OpenMMLab-Q** [comment]
Dear lianqing01,

First of all, we want to express our gratitude for your significant PR in the MMDetection3d project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

If you are Chinese or have **WeChat**，welcome to join our community on WeChat. You can add our assistant ：**openmmlabwx**. Please add **"mmsig + Github ID"** as a remark when adding friends：）

We would also like to invite you to join our Special Interest Group (SIG) private channel on **Discord,** where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. We look forward to seeing you there! Join us ：https://discord.gg/raweFPmdzG

Thank you again for your contribution❤

Best regards！@lianqing01

**Pull request merged**